python/pyfury/_serialization.pyx

```diff
@@ -1644,29 +1656,143 @@ cdef class CollectionSerializer(Serializer):
     cpdef int16_t get_xtype_id(self):
         return -FuryType.LIST.value
 
+    cpdef int8_t write_header(self, Buffer buffer, value):
```

Comment:
Currently golang didn't implement this protocol yet, could we enable it only for pure python serialization?

Reply:
`write_header` will only be called in `Python`, and the `Xlang` format has not been modified at the moment